references/cicd-pipeline/README.md

```diff
@@ -200,6 +200,14 @@ You are responsible to ensure you have the following plugins enabled:
 - [HTML Publisher](https://plugins.jenkins.io/htmlpublisher/)
 - [Cucumber Reports](https://plugins.jenkins.io/cucumber-reports/)
 
+The Jenkinsfile provided in this repo expects a credential object to exist for Edge deployments: 
```

Comment:
Suggestion not necessary: `Edge` => `Apigee Edge`? Otherwise Edge deployments might confuse people with deployment at the edge?